fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/SimpleObserver.java

```diff
@@ -68,6 +82,60 @@
     static final Integer EVENT_TYPES = NODE_ADDED + NODE_REMOVED + NODE_MOVED + PROPERTY_ADDED + PROPERTY_CHANGED
             + PROPERTY_REMOVED;
 
+    private static Stream<FedoraResource> getAllChildren(final FedoraResource resource) {
```

Comment:
Should this be a method on `FedoraResource`? Different impls may have important optimizations for it.